docs/user-guide.md

```diff
@@ -1189,15 +1189,24 @@ to load the ELF file over JTAG on Juno.
 The latest version of the AArch64 build of ARM Trusted Firmware has been tested
 on the following ARM FVPs (64-bit host machine only).
 
-*   `Foundation_Platform` (Version 10.2, Build 10.2.20)
-*   `FVP_Base_AEMv8A-AEMv8A` (Version 8.4, Build 0.8.8402)
-*   `FVP_Base_Cortex-A57x4-A53x4` (Version 8.4, Build 0.8.8402)
+NOTE: Unless otherwise stated, the model version is Version 11.0 Build 34.
```

Comment:
Please keep the full build number (presumably 11.0.34) as this is what is reported by launching the FVP with --version (see note below).